src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import Icon from "../Icons/Icon";
+import { IconNames } from "../Icons/IconTypes";
+
+export interface DropdownProps {
+  /** Inner label on the button that opens the dropdown */
+  dropdownLabel?: React.ReactNode;
+  /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
+  blockName?: string;
+  /** className you can add in addition to 'input' */
+  className?: string;
+  /** ID that other components can cross reference for accessibility purposes */
+  id?: string;
+  /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+  modifiers?: string[];
+}
+
+/** Dropdown component that shows content on toggle */
+export default function Dropdown(
+  props: React.PropsWithChildren<DropdownProps>
+) {
+  const {
+    modifiers = [],
+    blockName,
+    id,
+    className,
+    dropdownLabel,
+    children,
+  } = props;
+
+  return (
+    <div id={id} className={bem("dropdown", modifiers, blockName, [className])}>
+      <input
+        id={`dropdown-${id}`}
+        className={bem("input", modifiers, "dropdown")}
+        type="checkbox"
+      />
+      <label
+        htmlFor={`dropdown-${id}`}
+        className={bem("label-group", modifiers, "dropdown")}
+      >
+        <span className={bem("label", modifiers, "dropdown")}>
```

Comment:
`span`s are inline, so you won't be able to apply a margin to this in the way you're expecting to be able to. My recommendation would be to make the label group a flex group with this as a div so that

1. have the label contexts float next to each other with the new div
2. apply the margin to the label text
3. align-content so that the icons center in relation to the text

I think that will get us closer to the designs we're seeing in [the specs](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=17519%3A64445).